components/docs-chef-io/content/automate/ha_chef_backend_to_automate_ha.md

```diff
@@ -17,276 +15,334 @@ gh_repo = "automate"
-- Post Migration Customer can not login with chef-server users to Supermarket. 
+- Automate HA does not support supermarket authentication with chef-server user credentials.
+- Post Migration Customer can not log in with chef-server users to Supermarket.
+
 {{< /warning >}}
 
 This page explains the procedure to migrate the existing Standalone Chef Infra Server or Chef Backend data to the newly deployed Chef Automate HA. This migration involves two steps:
 
--   Back up the data from an existing Chef Infra Server or Chef Backend via `knife-ec-backup`.
--   Restore the backed-up data to the newly deployed Chef Automate HA environment via `knife-ec-restore`.
+- Back up the data from an existing Chef Infra Server or Chef Backend via `knife-ec-backup`.
+- Restore the backed-up data to the newly deployed Chef Automate HA environment via `knife-ec-restore`.
 
-Take backup using the `knife-ec-backup` utility and move the backup folder to the newly deployed Chef Server. Later, restore using the same utility. The backup migrates all the cookbooks, users, data-bags, policies, and organizations.
+Take a backup using the `knife-ec-backup` utility and move the backup folder to the newly deployed Chef Server. Later, restore using the same utility. The backup migrates all the cookbooks, users, data-bags, policies, and organizations.
 
-`knife-ec-backup` utility backups and restores the data in an Enterprise Chef Server installation, preserving the data in an intermediate, editable text format. It is similar to the knife download and knife upload commands and uses the same underlying libraries. It includes workarounds for unsupported objects by the tools and various Server API deficiencies. The goal is to improve knife download, knife upload, and the Chef Infra Server API to deprecate the tool.
+`knife-ec-backup` utility backups and restores the data in an Enterprise Chef Server installation, preserving the data in an intermediate, editable text format. It is similar to the knife download, uploads commands, and uses the same underlying libraries. It includes workarounds for unsupported objects by the tools and various Server API deficiencies. The goal is to improve knife download, knife upload, and the Chef Infra Server API to deprecate the tool.
 
 {{< note >}}
 
-- The migration procedure is tested on Chef Server version 14+.
-- The migration procedure is possible above Chef Backend version 2.1.0.
+- The migration procedure is tested on Chef Server versions 14+ and 15+.
+- The migration procedure is tested and is possible on the Chef Backend version above 3.1.1.
 
 {{< /note >}}
 
+## Prerequisites
+
+Check the [Prerequisites](/automate/ha_aws_deployment_prerequisites/#migration) page before migrating.
+
 ## Backup the Existing Chef Infra Server or Chef Backend Data
 
-1.   Execute below command to install Habitat:
+1. Execute the below command to install Habitat:
```

Comment:
Do we need to install tab while running `knife-ec-backup` on a Chef Server or Instance?

Reply:
This doc talks about installing using `ruby gem`
https://github.com/chef/knife-ec-backup#chef-infra-server-install-recommended

Reply:
@kalroy We have tested migration by installing hab and then knife-ec-backup package only